projects/news_feed/README.md

```diff
@@ -0,0 +1,16 @@
+# Sports News Feed
+
+## The purpose of this project is to inform its users about everydays sports events and news using a specially designed GUI (Graphical User Interface) system.
+
+You can check the status of project by clicking on [M-Shan TechCrunch projects](https://github.com/M-Shan/TechCrunch/projects)
+
+### How to access a website using python
+
+> The first step is to understand that how we can send URL requests using python. Luckily, Python comes with the package called **urllib** that simplifies this task. This package further contains five modules.
+1. Request
+1. Response
+1. Error
+1. Parse
+1. Robotparser
```

Comment:
good info but you could also check some high level python modules (pandas, beautifulsoup) for web page crawling.
check following links:

- https://www.edureka.co/blog/web-scraping-with-python/
- https://realpython.com/beautiful-soup-web-scraper-python/

these are just suggestions from my side you can decide what to use in future :)

Reply:
@M-Shan README.md files have been updated. I have read one of the links that you shared. They are quite interesting and helpful!